spot-client/src/common/css/_modal.scss

```diff
@@ -16,20 +16,23 @@
     }
 
     .modal-content {
+        background-color: var(--container-bg-color);
```

Comment:
Are we now trying to prevent buttons outside of the modal from being clickable? (We'll have to prevent keyboard navigation in the future if this is the case)

Reply:
Nevermind, I was somehow thinking modal-content was the modal shroud.